package.json

```
@@ -1,6 +1,9 @@
 {
   "name": "atat",
   "version": "1.0.0",
+  "engines": {
+    "node": "^14.16.0"
+  },
```

Could be make this more generic?
```suggestion
  "engines": {
    "node": ">=14.0.0 <15.0.0" 
  },
```

---

The version was tested for deployment and can not be changes.